### DISCUSSION TITLE
prisma init error

### DISCUSSION THREAD

**Discussion by saeedo-khan** [prisma/prisma#17836] [Q&A] [answered]
Downloading Prisma engines for Node-API for windows [                    ] 0%Error: request to 
https://binaries.prisma.sh/all_commits/ca7fcef713137fa11029d519a9780db130cca91d/windows/migration-engine.exe.gz.sha256 failed, reason: getaddrinfo ENOTFOUND 1080

~ I used to use proxy 127.0.0.1:1080 but I removed it

**Comment by janpio** [comment, marked as answer]
The URL works, so there is still something on your system blocking you from being able to access it.

**Reply by saeedo-khan** [reply]
yup, i discovered i added a proxy to system environment so I removed it and it worked well thnx for your tip appreciate

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
